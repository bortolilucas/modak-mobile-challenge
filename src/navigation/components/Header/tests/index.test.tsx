import * as states from '@/navigation/components/Header/tests/componentStates';
import { testSnapshots } from '@/testing/snapshots';

describe('Header', () => {
  testSnapshots(states);
});
