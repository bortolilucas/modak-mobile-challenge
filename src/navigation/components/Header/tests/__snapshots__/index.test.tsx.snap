// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match Default snapshot 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;

exports[`Header should match WithBackButton snapshot 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;
